libs/components/src/lib/alert/vl-alert.uig-css.ts

```diff
@@ -0,0 +1,10 @@
+import { css, CSSResult } from 'lit';
+
+const styles: CSSResult = css`
+.vl-alert__close {
+    /* revert overflow and position, these are not needed */
+    overflow: visible;
+    position: static;
```

Comment:
Zouden hier ook kunnen opteren voor [revert](https://developer.mozilla.org/en-US/docs/Web/CSS/revert), maar het leek me beter het expliciet te zetten.

Reply:
Misschien ook hier de uitleg even;
Bij DV wordt die knop absoluut gepositioneerd in zijn container.
Dat is denk ik nog een overblijfsel van vroeger (toen die nog geen flex was). 

De alert is nu echter wel een flex container:
icoon - content - button

Daardoor is die absolute positionering niet nodig lijkt me.